### PR TITLE
contrib: improved monitoring index with consistent spacing

### PR DESCRIPTION
**Pull request by vsusini**
Spacing was missing on flows monitor page. Ended up cleaning up the monitor page. 

- Added `MonitoringDisplay` to clean up duplicated code. 
- Added `MonitoringHeaderLink` to clean up duplicated code.
- Added consistent spacing to `MonitoringDisplay` while removing additional spacing on individual displays.

### CODE REVIEW

File: src/pages/monitoring/index.tsx [hunk 1]
```diff
@@ -19,33 +19,36 @@ import Flow from './flows';
 export default function Monitoring({ showFeesInPnl, ...pageProps }: { showFeesInPnl: boolean } & PageProps) {
   const poolInfo = usePoolInfo(pageProps.custodies);
 
+  type MonitorViews = 'lite' | 'full' | 'livePositions' | 'userProfiles' | 'allStaking' | 'flows';
```
Comment (adrena-orex) [inline src/pages/monitoring/index.tsx:22]: Could be nice to go a little farer and have:

```
type MonitorViews = 'lite' | 'full' | 'livePositions' | 'userProfiles' | 'allStaking' | 'flows';

const monitorViews: MonitorViews[] = ['lite', 'full', 'livePositions', 'userProfiles', 'allStaking', 'flows'] as const;

const monitorViewsNames: Record<MonitorViews, string> = {
  lite: 'Lite',
  full: 'Full',
  livePositions: 'Live Positions',
  userProfiles: 'User Profiles',
  allStaking: 'Staking',
  flows: 'Flows',
};
```

so you can loop later to create the `MonitoringHeaderLink`

File: src/pages/monitoring/index.tsx [hunk 2]
```diff
@@ -19,33 +19,36 @@ import Flow from './flows';
...
-          searchParamsView as string,
-        )
-      ) {
-        return;
-      }
-
-      setView(
-        searchParamsView as 'lite' | 'full' | 'livePositions' | 'userProfiles' | 'allStaking' | 'flows',
-      );
+    let searchParamsView = searchParams.get('view');
```
Comment (adrena-orex) [inline src/pages/monitoring/index.tsx:43]: could use:

```
    const searchParamsView = searchParams.get('view') ?? 'lite';
```

File: src/pages/monitoring/index.tsx [hunk 3]
```diff
@@ -219,91 +126,83 @@ export default function Monitoring({ showFeesInPnl, ...pageProps }: { showFeesIn
...
+
+  function MonitoringDisplay(view: MonitorViews, translateX: number, isFull: boolean = false) {
+    return <motion.div
+      initial={{
+        opacity: 0,
+        translateX: translateX,
+      }}
+      animate={{ opacity: 1, translateX: 0 }}
+      transition={{ duration: 0.3 }}
+      className={`min-h-[80vh] ${isFull ? "z-10" : ""}`}
```
Comment (adrena-orex) [inline src/pages/monitoring/index.tsx:181]: probably always could be `z-10`

**Comment by adrena-kino** [comment]
Hello @vsusini , thanks for the PR, much appreciated ! We updated the versions of libraries, hence having new eslint issues popping, can you fix these :

```
Failed to compile.
./src/pages/monitoring/index.tsx
49:9  Warning: The 'searchParams' object construction makes the dependencies of useEffect Hook (at line 56) change on every render. To fix this, wrap the initialization of 'searchParams' in its own useMemo() Hook.  react-hooks/exhaustive-deps
52:9  Error: 'searchParamsView' is never reassigned. Use 'const' instead.  prefer-const
info  - Need to disable some ESLint rules? Learn more here: https://nextjs.org/docs/app/api-reference/config/eslint#disabling-rules
Error: Command "npm run build" exited with 1
```

Then I will merge your contrib 👍

**Pull request merged**